FSubject: [PATCH] Better QS detail clip animation

Clip animation could be started with panel expanding.
Animation's target height should be expanded panel height.

Test: Click QQS which has detail view (ex. Battery tile)
 > Check animation while panel is expanding

diff --git a/packages/SystemUI/src/com/android/systemui/qs/QSDetailClipper.java b/packages/SystemUI/src/com/android/systemui/qs/QSDetailClipper.java
index a318efc..726abe5 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/QSDetailClipper.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/QSDetailClipper.java
@@ -40,8 +40,8 @@
         if (mAnimator != null) {
             mAnimator.cancel();
         }
-        final int w = mDetail.getWidth() - x;
-        final int h = mDetail.getHeight() - y;
+        final int w = mDetail.getMeasuredWidth() - x;
+        final int h = mDetail.getMeasuredHeight() - y;
         int innerR = 0;
         if (x < 0 || w < 0 || y < 0 || h < 0) {
             innerR = Math.abs(x);
